x-pack/plugins/security_solution/public/common/components/landing_page/get_started/card_item.tsx

```diff
@@ -91,7 +96,9 @@ const CardItemComponent: React.FC<{
       className={cardClassNames}
       hasBorder
       paddingSize="none"
-      css={cardItemPanelStyle}
+      css={css`
```

Comment:
for most of these, it looks like you are already calling css`stylestring` in the hook, you shouldn't have to do it again. If those hooks just returned normal strings, we should also avoid calling css directly in the render as well: https://engineering.deptagency.com/optimizing-emotion-in-react

Reply:
I saw some error with the original implementation, haven't got a better way to fix this. The error I saw was:
```you have tried to stringify object returned from css function. It isn't supposed to be used directly, but rather handed to emotion so it can handle it (e.g. as value of css prop)```

Reply:
Looks like it doesn't like mixing className and css together. After changing import to emotion/css, it's working fine.